apps/google-analytics-4/frontend/src/helpers/signed-requests.ts

```diff
@@ -0,0 +1,38 @@
+import { CreateAppSignedRequestProps } from 'contentful-management/dist/typings/entities/app-signed-request';
+import { PlainClientAPI } from 'contentful-management/dist/typings/plain/common-types';
+
+async function makeSignedRequest(
```

Comment:
Small naming thing but when I look at this function name and what the function does it does not make sense. I expect this to "make" (i.e. return) a signed Request. Instead We're actually _doing_ the signed request.

I'd suggest something like `fetchWithSignedRequest` or something that makes it clear that this fires off a (fetch) request.

Reply:
done